### PR TITLE
(B) QTY-9035: Changing callback to after create

### PR DESCRIPTION
**Pull request by jothankele**
[QTY-9035](https://strongmind.atlassian.net/browse/QTY-9035)

## Purpose 
Course updates were causing course conclude dates to changes dates depending on their timezone.

## Approach 
Pass param in frontend to manually calculate timezones

## Testing
Rspec, Local testing on teamviewer VM

[QTY-9035]: https://strongmind.atlassian.net/browse/QTY-9035?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/decorators/models/course_decorator.rb [hunk 1]
```diff
@@ -106,6 +106,7 @@ def set_course_start_end_time_from_school
 
     self.start_at = course_start_time_from_school
     self.conclude_at = course_end_time_from_school
+    self
```
Comment (adiaz04) [inline app/decorators/models/course_decorator.rb:109]: you may need to save here since it's an `after_create`

Comment (jothankele) [reply app/decorators/models/course_decorator.rb:109]: changed to `before_create`

**Review by tylerCaineRhodes — CHANGES_REQUESTED** [review]
Looks good, although I just want y'all to check my draft PR first, in case we can simplify the call/specs in any way.

**Comment by jothankele** [comment]
> Looks good, although I just want y'all to check my draft PR first, in case we can simplify the call/specs in any way.

We implemented a compromise from the draft PR and are proceeding since Tyler is outageing.

**Pull request merged**